ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/_cq_dialog/.content.xml

```diff
@@ -281,12 +279,13 @@
                               jcr:primaryType="nt:unstructured"
                               sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
                               name="./tooltipVisible"
-                              text="Always show short description"
+                              text="Show tooltip below the component"
```

Comment:
we should not assume the placement of tooltip.
it should be "Always show the tooltip"